README.md

```diff
@@ -78,17 +82,23 @@ The above code will return the following:
 19012600725504
 ```
 
+### Charts
+
 You can also ask PandasAI to draw a graph:
 
 ```python
 pandas_ai(
     df,
-    "Plot the histogram of countries showing for each the gpd, using different colors for each bar",
+    "Plot the histogram of countries showing for each the gdp, using different colors for each bar",
 )
 ```
 
 ![Chart](images/histogram-chart.png?raw=true)
 
+You can save any charts generated by PandasAI by setting the `save_charts` parameter to `True` in the `PandasAI` constructor. For example, `PandasAI(llm, save_charts=True)`. Charts are saved in `./pandasai/exports/charts` .
```

Comment:
Saving the charts in the package folder might make them difficult to find for some users, especially when PandasAI is installed into a virtual environment.
Alternatively we could display the save path or/and allow the user to specify a path

Reply:
Yeah, printing the newly generated path is a great idea, let's go for it!